pom.xml

```diff
@@ -6,7 +6,7 @@
 	<parent>
 		<groupId>org.springframework.cloud</groupId>
 		<artifactId>spring-cloud-build</artifactId>
-		<version>2.0.0.M6</version>
+		<version>2.0.0.RC1</version>
```

Comment:
This is wrong. There is no `spring-cloud-build:2.0.0.RC1` yet: https://github.com/spring-cloud/spring-cloud-build/releases

Reply:
Oh! Then I guess we can't really move to SB2 RC1 until there is an actual `spring-cloud-build:2.0.0.RC1`? :(

Is there any rough estimate for that?

Reply:
I think you talk about this: https://github.com/spring-cloud/spring-cloud-release/milestone/37

Reply:
I still think this is wrong. See the link I've mentioned first. Plus pay attention to the Travis report.

Reply:
Will there be a `spring-cloud-build` RC1 release?

I was assuming there would be, because there was an M6 one.

> Seems like the Milestones repo doesn't have RC1 yet, so we'll have to wait a little bit to merge.

I added `spring-cloud-dependencies` as a BOM, but we still need a parent for these files.

Reply:
> I was assuming there would be, because there was an M6 one.

No, Spring Cloud and Spring Boot are different projects, although the first one depends on the second. But still it doesn't mean that they have to be released at the same day. I've pointed you to the `Finchley.M6` which is now planned for Feb 13 (disappointing 😢 ): https://github.com/spring-cloud/spring-cloud-release/milestone/37.

If we don't have chances to wait until there, we don't have choice unless rollback all the versions to the proper milestones and release as is, without any Spring Boot 2.0 RC1 goals.